src/server/utilities/cspHeader/index.js

```diff
@@ -280,9 +287,23 @@ const directives = {
       'https://news.files.bbci.co.uk', // STY include styles
       'https://platform.twitter.com', // Social Embeds
       'https://ton.twimg.com', // Social Embeds
+      'https://news.files.bbci.co.uk', // STY includes
+      'https://news.test.files.bbci.co.uk', // STY includes
+      'https://static.bbc.co.uk', // STY include
       "'unsafe-inline'",
     ],
   },
+  fontSrc: {
+    amp: [
+      'https://gel.files.bbci.co.uk', // STY includes
+      'https://ws-downloads.files.bbci.co.uk', // STY includes
+    ],
+    canonical: [
+      'https://gel.files.bbci.co.uk', // STY includes
+      'https://ws-downloads.files.bbci.co.uk', // STY includes
```

Comment:
Small suggestion regarding comments. 
```suggestion
      'https://gel.files.bbci.co.uk', // Reith fonts
      'https://ws-downloads.files.bbci.co.uk', // Other WS fonts
    ],
    canonical: [
      'https://gel.files.bbci.co.uk', // Reith fonts
      'https://ws-downloads.files.bbci.co.uk', // Other WS fonts
```